src/FluentValidation.Tests/ValidatorTesterTester.cs

```diff
@@ -19,25 +19,30 @@
 #endregion
 
 namespace FluentValidation.Tests {
-	using System;
 	using System.Collections.Generic;
 	using System.Linq;
 	using Xunit;
 	using TestHelper;
-	using Validators;
 	using System.Threading.Tasks;
 
 	public class ValidatorTesterTester {
 		private TestValidator validator;
+		private const string sutSameForenameAndSurnameMessage = "A person cannot have the same forename and surname! What the hell, Helle Helle! https://en.wikipedia.org/wiki/Helle_Helle";

```

Comment:
Can you remove the "what the hell" from the message please - language needs to be professional. A simple message like "Model level test message" is sufficient.